ketcher-autotests/tests/File-Management/Open-And-Save-Files/Rxn-Files/rxn-files.spec.ts

```diff
@@ -1,12 +1,278 @@
+  await dragMouseTo(xCoordinatesWithShift, y, page);
+  await savedFileInfoStartsWithRxn(page);
+
+  await pressButton(page, 'Cancel');
+  await selectLeftPanelButton(LeftPanelButton.ReactionPlusTool, page);
+  await page.mouse.click(xCoordinatesWithShiftHalf, yCoordinatesWithShift);
+  const ySecondChain = yCoordinatesWithShift + yDelta50;
+  await selectLeftPanelButton(LeftPanelButton.Chain, page);
+  await page.mouse.move(x, ySecondChain);
+  await dragMouseTo(xCoordinatesWithShift, ySecondChain, page);
+  await savedFileInfoStartsWithRxn(page);
+
+  await pressButton(page, 'Cancel');
+  await selectLeftPanelButton(LeftPanelButton.Erase, page);
+  await page.mouse.click(xCoordinatesWithShiftHalf, yCoordinatesWithShift);
+  await selectNestedTool(page, ArrowTool.ARROW_FILLED_BOW);
+  const yArrowStart = y + yDelta20;
+  const yArrowEnd = yArrowStart + yDelta20;
+  await page.mouse.move(xCoordinatesWithShiftHalf, yArrowStart);
+  await dragMouseTo(xCoordinatesWithShiftHalf, yArrowEnd, page);
+  await savedFileInfoStartsWithRxn(page, true);
+
+  await pressButton(page, 'Cancel');
+  await selectTopPanelButton(TopPanelButton.Clear, page);
+  await selectNestedTool(page, ArrowTool.ARROW_FILLED_BOW);
+  await page.mouse.move(xCoordinatesWithShiftHalf, yArrowStart);
+  await dragMouseTo(xCoordinatesWithShiftHalf, yArrowEnd, page);
+  await savedFileInfoStartsWithRxn(page, true);
+});
+
+// EPMLSOPKET-8904 - TO DO
+// TO DO: looks like issue was not fixed
+// Need to recheck issue #1837, added new screenshot there
+test.fixme(
+  'Open and Save file - Structure is not missing when "Paste from clipboard" or "Open from file" if reaction consists of two or more reaction arrows and structures',
+  async ({ page }) => {
+    /**
+     * Test case: EPMLSOPKET-8904
+     * Description: Structure isn't missing when "Paste from clipboard" or "Open from file" if reaction consists of two or more reaction arrows and structures
+     */
+    await page.goto('');
+    const RING_OFFSET = 150;
+    const ARROW_OFFSET = 20;
+    const ARROW_LENGTH = 100;
+    await drawReactionWithTwoBenzeneRings(
+      page,
+      RING_OFFSET,
+      ARROW_OFFSET,
+      ARROW_LENGTH,
+    );
+
+    const xOffsetFromCenter = 50;
+    await selectNestedTool(page, ArrowTool.ARROW_FILLED_BOW);
+    await moveMouseToTheMiddleOfTheScreen(page);
+    await clickOnTheCanvas(page, xOffsetFromCenter, 0);
+    const expectedFileV2000 = await getRxn(page);
+    await takeEditorScreenshot(page);
+    await saveToFile(
+      'RXN/structure-with-two-reaction-arrows-v2000-saved.rxn',
+      expectedFileV2000,
+    );
+    const expectedFileV3000 = await getRxn(page);
+    await saveToFile(
+      'RXN/structure-with-two-reaction-arrows-v3000-saved.rxn',
+      expectedFileV3000,
+    );
+    await selectAction(TopPanelButton.Clear, page);
+    await openFileAndAddToCanvas(
+      'RXN/structure-with-two-reaction-arrows-v3000-saved.rxn',
+      page,
+    );
```

Comment:
Don't we need to check the same for v2000?

Reply:
We have this check for v2000 file in s-group tests, added it here too